README.md

```diff
@@ -170,6 +170,23 @@ You can install it through your favorite plugin manager:
 
   </details>
 
+## Configuration
+
+This module accepts the following configuration with the shown defaults:
+
+```lua
+{
+    ...
+    ["core.integrations.telescope"] = {
+        insert_file_link = {
+            -- Whether to show the title preview in telescope. Affects performance with a large
```

Comment:
from the top of my head I can't say rn what exactly this does
I'll look at this again 2moro and perhaps it's self explanatory if you're familiar with the picker
so you can just ignore this for now